CMakeLists.txt

```diff
@@ -35,8 +35,10 @@ ENDIF ()
 IF (MQTT_USE_WS)
     MESSAGE (STATUS "WebSocket enabled")
     SET (CMAKE_CXX_FLAGS "-DMQTT_USE_WS ${CMAKE_CXX_FLAGS}")
+    SET (BOOSTVERSION "1.66.0")
 ELSE ()
     MESSAGE (STATUS "WebSocket disabled")
+    SET (BOOSTVERSION "1.59.0")
```

Comment:
I'm not sure this is accurate anymore.

There were bigger changes that cause incompatibilities with versions of boost older than 1.67.

But if this does actually work, that'd be great to know about.

Reply:
Ohh, you are right. This will be reverted.